Offroad-iOS/Offroad-iOS/Global/Extensions/UIScreen+.swift

```diff
@@ -13,4 +13,20 @@ extension UIScreen {
     static var current: UIScreen {
         UIWindow.current.screen
     }
+    
+    /// 현재 screen의 사이즈를 반환하는 타입 변수
+    static var currentScreenSize: CGSize {
+        current.bounds.size
+    }
+    
+    /// screen의 긴 변의 길이가 짧은 변의 길이의 두 배가 넘는 지의 여부를 계산하는 변수.
+    /// 사용 중인 기기가 홈 버튼이 있는 기종인지 여부를 판단하기 위해 사용할 수 있음.
+    var isAspectRatioTall: Bool {
+        let isVerticalDirection: Bool = bounds.size.height > bounds.size.width
```

Comment:
P3

24~30번째 줄 max()와 min() 함수 이용하면 더 간단하게 구현할 수 있을 것 같습니다.
```suggestion
       var isAspectRatioTall: Bool {
        return max(bounds.size.width, bounds.size.height) >= min(bounds.size.width, bounds.size.height) * 2
    }
```
참고하시고 더 편한 방법으로 구현해주시면 좋겠습니다:)

Reply:
좋은 것 같습니다! 코드가 훨씬 간결하고 직관적일 것 같습니다!